src/handler/SpecifyServiceUrlStep.ts

```diff
@@ -25,6 +26,7 @@ export class SpecifyServiceUrlStep implements IStep {
         if (projectMetadata.serviceUrl === undefined) {
             throw new OperationCanceledError("Service URL not specified.");
         }
+        projectMetadata.bootVersions = await serviceManager.getBootVersions(projectMetadata.serviceUrl);
```

Comment:
you don't do this. it's "specify url" step here. just let it check in "specify boot version" step.